api/web3server_marshal.go

```diff
@@ -75,6 +80,28 @@ type (
 		CurrentBlock  string `json:"currentBlock"`
 		HighestBlock  string `json:"highestBlock"`
 	}
+	structLog struct {
```

Comment:
if custom marshalling is used?

Reply:
Yes, `structLog` = `logger.StructLog`, but `logger.StructLog` ignored json field `storage`, web3 API needs to export it

Reply:
it could be done in `MarshalJSON()`

Reply:
I  think there is no need to implement `marshalJSON` here, the `structLog` field can already be automatically marshaled inside. use `interface{}` is harder to understand